### ISSUE TITLE
pytoml dependency is unmaintained

### ISSUE THREAD

**Issue by duckinator** [pypa/pep517#61]
According to [pyproject.toml](https://github.com/pypa/pep517/blob/master/pyproject.toml), this library depends on pytoml.

However, [pytoml is explicitly unmaintained](https://github.com/avakar/pytoml#deprecated) and [lacks support for dotted keys](https://github.com/avakar/pytoml/issues/38).

It'd be great if this switched a still-supported toml library. I've found [uiri/toml](https://github.com/uiri/toml) works well.

---

Context on how I discovered this:

When using [bork](https://github.com/duckinator/bork) (which relies on this library), I tried putting this in `pyproject.toml`:

```
[tool.bork]
zipapp.enabled = true
zipapp.main = "bork.cli:main"
```

but it results in a hilariously vague error message of `pyproject.toml(8, 1): msg` (that's it).

I wound up having to do this instead:

```
[tool.bork.zipapp]
enabled = true
main = "bork.cli:main"
```

**Comment by duckinator** [comment]
Good news: I think the APIs are identical, or at least very close.

The only potential problem would be handling of unicode on Python 2: https://github.com/pypa/pep517/blob/11a129044479bb68699b815fce01877b5cc17c88/pep517/wrappers.py#L220-L225

I'll look into making a PR switching from pytoml -> toml later today.

**Comment by takluyver** [comment]
Thanks! Obviously you'll need any other projects that read your `pyproject.toml` to make a similar switch.

Closed by #62.

**Comment by pradyunsg** [comment]
@takluyver Any idea when we'd have a release with this? pip would need to update its vendored dependencies for this change, so just wanna make sure.

**Comment by takluyver** [comment]
I've released 0.7.0 including this change just now.

**Comment by duckinator** [comment]
thanks!
